src/Juvix/Compiler/Core/Pretty/Base.hs

```diff
@@ -183,6 +224,28 @@ instance PrettyCode Node where
     Closure env l@Lambda {} ->
       ppCode (substEnv env (NLam l))
 
+instance PrettyCode Stripped.Node where
```

Comment:
can we have an embedding from `Stripped.Node` to `Node` and then just pretty print `Node`?

Reply:
The issue with converting the InfoTable back could be that you forget some information and need to provide some defaults which might not make much sense.

Reply:
I see, then it makes more sense to leave it as it is